src/scenic/core/vectors.py

```diff
@@ -362,7 +368,7 @@ def localAnglesFor(self, orientation) -> typing.Tuple[float, float, float]:
         That is, considering ``self`` as the parent orientation, find the Euler angles
         expressing the given orientation.
         """
-        local = orientation * self.inverse
+        local = self.inverse * orientation
```

Comment:
This formula isn't obvious, so let's add a unit test for this function (in `tests/core/test_vectors.py`). The following works:
```
parent = Orientation.fromEuler(pi/4, pi/4, 0)
target = Orientation.fromEuler(-3*pi/4, pi/4, pi/4)
angles = parent.localAnglesFor(target)
assert angles == pytest.approx((-3*pi/4, pi/2, 0))
```
By waving my hands around I can convince myself the resulting angles do in fact rotate the parent onto the target. Actually we might as well test that relationship more robustly:
```
for i in range(100):
    parent = Orientation.fromEuler(*[random.uniform(-pi, pi) for _ in range(3)])
    target = Orientation.fromEuler(*[random.uniform(-pi, pi) for _ in range(3)])
    local = Orientation.fromEuler(*parent.localAnglesFor(target))
    assert target.approxEq(parent * local)
```

Reply:
Good idea! Test added.

Reply:
Can you please add the first test too? It's not subsumed by the second test since it's checking the specific angles returned, not just the round-trip.

Reply:
Whoops, sure thing.